apps/web/src/pages/activities/ActivitiesPage.tsx

```diff
@@ -100,8 +99,7 @@ export function ActivitiesPage() {
   };
 
   return (
-    <PageContainer>
-      <PageMeta title="Activity Feed" />
```

Comment:
The `PageContainer` has a prop `title` that is basically rendering `PageMeta` inside. Did the changes across the code.